### DISCUSSION TITLE
Database very long to open

### DISCUSSION THREAD

**Discussion by smutel** [keepassxreboot/keepassxc#8060] [Q&A]
Hello,

I have one database witch is very long to open. Sometimes I received a message from the system: This application does not respond, kill or wait".

Is-it possible to activate some log files to see what's wrong ? I would like to check if it is due to my database, the browser integration or something else.

Thanks.

**Comment by phoerious** [comment]
Did you change the encryption settings? Long open times sound like the number of key derivation rounds is too high for your system. After opening the database go to Database -> Database Settings -> Security -> Encryption Settings. Then check the "Advanced Settings" box in the lower left corner and report what encryption settings you have.

**Reply by smutel** [reply]
Your are right.

In one database (open is quickly), I have these settings:
- AES 256-bit
- AES-KDF (KDBX 4)
- 1000000

In the other (long to open), I have these settings:
- AES 256-bit
- Argon2d (KDBX 4 - recommended)
- 10
- 64Mio
- 2fils d'exécution

What is the best parameter for security and performance ?

**Reply by droidmonkey** [reply]
Just click the one second benchmark button. That will already be overkill for security.

**Reply by smutel** [reply]
I setup the second database with the same config (AES 256-bit / AES-KDF (KDBX 4) / 1000000).
However the database is still long to open.

**Reply by droidmonkey** [reply]
What is the size of the database?

**Reply by smutel** [reply]
71,9 kB
